tour/authorizations.md

```diff
@@ -1,46 +1,49 @@
 ---
 title: Authorizations
 ---
-Authorizations are a set of Strings that enable a user to read protected data. A column visibility is a boolean expression 
+
+[Authorizations] are a set of Strings that enable a user to read protected data. A [ColumnVisibility] is a boolean expression
 that is evaluated using the authorizations provided by a scanner. If it evaluates to true, then the data is visible. 
 
 For example:
-* Bob scans with authorizations = { IT, User }
-* Tina scans with authorizations = { Admin, IT, User }
-* Row1:family1:qualifier1 has Visibility = { Admin && IT && User }
-* Bob will **not** see Row1:family1:qualifier1
-* Tina will see Row1:family1:qualifier1
+* Bob has authorizations `IT, User`
+* Tina has authorizations `Admin, IT, User`
+* The key `row1:family1:qualifier1` has visibility `Admin && IT && User`
+* Bob will **not** see `row1:family1:qualifier1`
```

Comment:
Here we could say "Bob scans with authorizations { IT, User } but will not see...".  "Tina scans with authorizations { Admin, IT , User } and will see ..."

Reply:
Fixed in ee62cbca